### PR TITLE
Documentation: add a group query example for the ldap connector.

### PR DESCRIPTION
**Pull request by rithujohn191**
A lot of people seem to be confused about how they should implement group specific queries. Thought it would be nice to have this as part of our docs.

### CODE REVIEW

**Comment by rithujohn191** [comment]
@ericchiang do you want me to replace `user DN` with just `DN`?

**Comment by rithujohn191** [comment]
Added an LDIF object example to clarify "DN". Also moved the example to the config section.

File: Documentation/ldap-connector.md [hunk 1]
```diff
@@ -191,6 +191,32 @@ groupSearch:
...
+
+```yaml
+groupSearch:
+  # BaseDN to start the search from. It will translate to the query
+  # "(&(objectClass=group)(member=<user DN>))".
+  baseDN: cn=groups,cn=compat,dc=example,dc=com
+  # Optional filter to apply when searching the directory.
+  filter: "(objectClass=group)"
+
+  userAttr: dc=example,dc=com # Use "DN" here not "uid"
```
Comment (ericchiang) [inline Documentation/ldap-connector.md]: This needs to be

```
userAttr: DN
```

**Comment by rithujohn191** [comment]
@ericchiang any additional change suggestions?

**Pull request merged**